drivers/homeless_summary_report/app/models/homeless_summary_report/client.rb

```diff
@@ -22,14 +22,21 @@ class Client < GrdaWarehouseBase
     ].freeze
     DEMOGRAPHIC_VARIANTS = [
       :all,
-      :mid_east_n_african,
-      :hispanic_latinaeo,
-      :black_african_american,
+      :am_ind_ak_native,
+      :am_ind_ak_native_hispanic_latinaeo,
       :asian,
-      :american_indian_alaskan_native,
-      :native_hawaiian_other_pacific_islander,
+      :asian_hispanic_latinaeo,
+      :black_af_american,
+      :black_af_american_hispanic_latinaeo,
+      :hispanic_latinaeo,
+      :mid_east_n_african,
+      :mid_east_n_african_hispanic_latinaeo,
+      :native_hi_pacific,
+      :native_hi_pacific_hispanic_latinaeo,
       :white,
+      :white_hispanic_latinaeo,
       :multi_racial,
+      :multi_racial_hispanic_latinaeo,
```

Comment:
Not sure if we can easily pull these from `HudUtility2024.race_ethnicity_combinations`, but if we can, we should.